### DISCUSSION TITLE
500 error when searching for users or posts from other instances

### DISCUSSION THREAD

**Discussion by robgitnyc** [mastodon/mastodon#21456] [General]
Hello.
Just instaled my Mastodon instance and it is working in general, but when using the search function to find users or posts from other instances i get a "500 internal server error" in the lower left corner.  How can I fix this?

Help will be much appreciated.

**Comment by robgitnyc** [comment]
error was ```Errno::EACCES (Permission denied @ dir_s_mkdir - /home/mastodon/live/public/system``` and was solved with ```chown -R mastodon:mastodon /home/mastodon/live/public```

**Reply by NicolaeNMV** [reply]
Cool glad you solved it :+1:

**Comment by brandonhines** [comment]
Where did you track down your error?

I'm having the same issue, but when I run `sudo journalctl -u mastodon-web` the results are empty.

**Reply by robgitnyc** [reply]
Run same command but with mastodon-* instead. Add -f for live log output.

```sudo journalctl -u mastodon-* -f```
